src/verifier.cc

```diff
@@ -113,8 +245,43 @@ Result Verifier::Probe(const ProbeCommand* command,
   return {};
 }
 
-Result Verifier::ProbeSSBO(const ProbeSSBOCommand*) {
-  return Result("Verifier::ProbeSSBO Not Implemented");
+Result Verifier::ProbeSSBO(const ProbeSSBOCommand* command,
+                           size_t size,
```

Comment:
Instead of passing a size and a const void*, passing a std::vector<uint8_t>* That will give you both data and size and it will get auto-cleanedup

Reply:
Sorry for making you confused. `size` here means the size of the ssbo in bytes. I don't know the number of elements at this moment. `cpu_memory` is just a pointer to ssbo. In my personal opinion, we would better not make a vector for this.